connector/connect/server/src/main/scala/org/apache/spark/sql/connect/planner/SparkConnectPlanner.scala

```diff
@@ -1447,7 +1447,7 @@ class SparkConnectPlanner(val session: SparkSession) {
       // No imported Python libraries
       Lists.newArrayList(),
       pythonExec,
-      "3.9", // TODO(SPARK-40532) This needs to be an actual Python version.
```

Comment:
will create a separate PR to remove these

Reply:
Ack!